grouper-to-github.py

```diff
@@ -49,14 +50,34 @@ def grouper_get_group_members(auth, group):
         data['WsGetMembersLiteResult']['wsSubjects']))
     return members
 
+def gh_account_check(auth, user):
+    '''Check if user's github.b.e account is active (i.e., have the logged in to github.b.e).'''
+    uri = '{}/users/{}'.format(config['github_base_uri'], user)
+    r = requests.get(uri, auth=auth)
+    if r.status_code == 404:
+        return False
+    else:
+        return True
+    
```

Comment:
Might be simpler to `return r.status_code != 404`.

Reply:
Thanks; good points. Funny, the verbosity of that return is something I would generally point out in looking over student code ...